frontend/src/app/components/circuitos/circuitos.component.ts

```diff
@@ -110,6 +112,12 @@ export class CircuitosComponent implements OnInit {
         bimesterPayment.validated = payment.validated;
         bimesterPayment.amount += this.newPayment.amount;
         bimesterPayment.complete = this.maintenanceFee <= bimesterPayment.amount;
+        const files = this.newPayment.files;
+        if (files) {
+          for (let i = 0; i < files.length; i++) {
```

Comment:
prefer always the "for each" syntax over the traditional for sintax

Reply:
when I wrote this code i tried to use foreach method or map method but i think it’s no available or isn't iterable to files